src/ErrorHandler/ThrowableRendererInterface.php

```diff
@@ -13,17 +13,21 @@ interface ThrowableRendererInterface
      * Convert throwable into its string representation suitable for displaying in production environment
      *
      * @param \Throwable $t
+     * @param string $template
+     * @param string|null $customPath
      * @return string
      */
-    public function render(\Throwable $t): string;
+    public function render(\Throwable $t, string $template = 'error', string $customPath = null): string;
```

Comment:
That would break all other renderers: JsonRenderer, PlainTextRenderer, XmlRenderer. Also, there's no need for template and path for all these. It is HTML-specific.

Reply:
I've moved providing custom path to config file to not change interface methods